extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -25,7 +25,7 @@ define('ext.wikia.adEngine.video.uapVideo', [
 		// On mercury splitLayoutVideoPosition and videoPlaceholderElement will be empty
 		// because we always display video in the same way there.
 		if (params.splitLayoutVideoPosition && params.videoPlaceholderElement) {
-			width = params.videoPlaceholderElement.width;
+			width = params.videoPlaceholderElement.offsetWidth;
```

Comment:
This small change removes sometimes a perception of a gap between first and second part of the ad, right?